client/src/components/ItemCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Instrument } from "@/interfaces/instrument";
+import { Musician } from "@/interfaces/musician";
+import classNames from "classnames";
+import Image from "next/image";
+import { Typography } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+
+interface Props {
+  item: Musician | Instrument;
+  onClick: (item: Musician | Instrument) => void;
+}
+
+const useStyles = makeStyles(() => ({
+  card: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    marginTop: "5%",
+    marginLeft: "2%",
+  },
+  selected: {
+    boxShadow: "0 0 0 5px #37A8FA",
+  },
+  image: {
+    width: 200,
+    height: 200,
+    overflow: "hidden",
+    borderRadius: "50%",
+    display: "flex",
+    justifyContent: "center",
+  },
+}));
+
+export const ItemCard = ({ item, onClick }: Props) => {
+  const handleClick = () => {
+    onClick(item);
+  };
+  const classes = useStyles();
+  return (
+    <div className={classNames(classes.card)}>
```

Comment:
Curious about the syntax here - what is the difference between using `classNames(classes.card)` and simply putting `classes.card `